libs/components/src/lib/button/button.component.ts

```diff
@@ -1,4 +1,3 @@
-// tslint:disable:no-host-metadata-property
```

Comment:
why is `eslint` not complaining about that?

Reply:
@michaelwittwer 
Since we allowed that in our ng-config
https://github.com/shiftcode/sc-commons-public/blob/main/packages/eslint-config-recommended/src/ng-config/index.ts